samples/routing/fair-distribution/Core_6/Client/Program.cs

```diff
@@ -83,9 +83,9 @@ static void AddRouting(EndpointConfiguration endpointConfiguration)
         #region Routing
 
         const string server = "Samples.FairDistribution.Server";
-        var routing = endpointConfiguration.Routing();
+        var routing = endpointConfiguration.UseTransport<MsmqTransport>().Routing();
         routing.RouteToEndpoint(typeof(PlaceOrder), server);
-        routing.Mapping.Physical.Add(
+        routing.GetSettings().GetOrCreate<EndpointInstances>().Add(
```

Comment:
> we should bite the bullet and add the file-based mapping. In real life that's what users would do.

If that's more realistic then I'm :+1:, even if that will make the sample a bit more complicated.